lms/templates/learner_dashboard/program_details_view.underscore

```diff
@@ -0,0 +1,9 @@
+<header class="program-header">
+	<div>
+		<h2 class="hd-2">Program Title</h2>
+		<p>Program Subtitle</p>
+	</div>
```

Comment:
I missed the dreaded tab characters here. Let's clean these up the next time we're in here.